tessera-core/src/test/java/com/quorum/tessera/transaction/EncodedPayloadManagerImplTest.java

```diff
@@ -192,30 +197,27 @@ public void decryptBadCipherText() {
 
         final String singleRecipientBase64 = "QfeDAys9MPDs2XHExtc84jKGHxZg/aj52DTh0vtA3Xc=";
         final PublicKey singleRecipient = PublicKey.from(Base64.getDecoder().decode(singleRecipientBase64));
-        final List<PublicKey> recipients = List.of(singleRecipient); //list the keys multiple times
-
-        final EncodedPayload samplePayload = EncodedPayload.Builder.create()
-            .withSenderKey(sender)
-            .withRecipientKeys(recipients)
-            .withCipherText(testPayload.getBytes())
-            .withPrivacyMode(PrivacyMode.STANDARD_PRIVATE)
-            .withAffectedContractTransactions(emptyMap())
-            .withExecHash(new byte[0])
-            .build();
-
-        when(messageHashFactory.createFromCipherText(any())).thenReturn(new MessageHash("test hash".getBytes()));
+        final List<PublicKey> recipients = List.of(singleRecipient); // list the keys multiple times
```

Comment:
here it is only once, not multiple times

Reply:
done